File: Problems/1108/main.py
def defangIPaddr(address):
  return address.replace('.', '[.]')


# TEST SUITE

# TEST 1

def test1():
  actual = defangIPaddr('1.1.1.1')
  expected = '1[.]1[.]1[.]1'
  if actual == expected:
    return '$$$$$'
  return 'X'

print(test1())

# TEST 2

def test2():
  actual = defangIPaddr('255.100.50.0')
  expected = '255[.]100[.]50[.]0'
  if actual == expected:
    return '$$$$$'
  return 'X'

print(test2())

File: Problems/1108/main.js
/**
 * @param {string} address
 * @return {string}
 */
const defangIPaddr = (address) => {
  return address.replace(/\./g, '[.]');
};

// TEST SUITE

// TEST 1

function test1() {
  const actual = defangIPaddr('1.1.1.1');
  const expected = '1[.]1[.]1[.]1';
  if (actual === expected) {
    return '$$$$$';
  }
  return 'X';
}

console.log(test1());

// TEST 2

function test2() {
  const actual = defangIPaddr('255.100.50.0');
  const expected = '255[.]100[.]50[.]0';
  if (actual === expected) {
    return '$$$$$';
  }
  return 'X';
}

console.log(test2());
